lua/neotest/lib/process/init.lua

```diff
@@ -1,16 +1,51 @@
 local nio = require("nio")
 
+--- Base module definition
 local neotest = { lib = {} }
+
 ---@toc_entry Library: Processes
 ---@text
 --- Utilities to run external processes easily.
 --- More complex use cases should use the lower level jobstart or vim.loop.spawn
 ---@class neotest.lib.process
 neotest.lib.process = {}
 
+--- @return table<string,string>
+local function base_env()
+  --- @type table<string,string>
+  local env = vim.fn.environ()
```

Comment:
Use `vim.uv.os_environ()` instead to avoid using a vimscript function (means we don't need to call the scheduler)